cardano-api/src/Cardano/Api/Governance/Poll.hs

```diff
@@ -76,29 +90,63 @@ instance HasTypeProxy GovernancePoll where
     data AsType GovernancePoll = AsGovernancePoll
     proxyToAsType _ = AsGovernancePoll
 
+instance AsTxMetadata GovernancePoll where
+    asTxMetadata GovernancePoll{govPollQuestion, govPollAnswers, govPollNonce} =
+      makeTransactionMetadata $ Map.fromList
+        [ ( metadataLabel
+          , TxMetaMap $
+           [ ( TxMetaNumber 0, metaTextChunks govPollQuestion )
```

Comment:
Did you choose numbers for the map keys to save space? I imaged the metadata being rendered on an explorer with `Text` keys e.g "govPollQuestion", "govPollAnswers" etc

Reply:
Yes and no. I chose numbers because there's no reason to chose something else. It's a binary format not meant to be human-readable to begin with; so whatever is the most compact makes sense here.